cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -103,6 +109,135 @@ function getGooglePayConfig() {
   };
 }
 
+function removeGiftCard() {
+  $.ajax({
+    type: 'POST',
+    url: 'Adyen-CancelPartialPaymentOrder',
+    data: JSON.stringify(store.partialPaymentsOrderObj),
+    contentType: 'application/json; charset=utf-8',
+    async: false,
+    success: function success(res) {
+      store.partialPaymentsOrderObj = null;
+
+      if (res.resultCode === 'Received') {
```

Comment:
Maybe its good this to be added to a constant file?

Reply:
It already is! constants has a `RESULTCODES.RECEIVED` field

Reply:
this is just the autogenerated file, we will make sure it's up to date before we release
This change was already made in the original file inside /src